src/schema.cr

```diff
@@ -98,7 +98,7 @@ module Cql
     # Schema.define
     # ```
     def build
-      @tables.each do |name, table|
+      @tables.each do |_tbl_name, table|
```

Comment:
This variable is unused, as indicated by Ameba. I've also renamed it from `_name` to `_tbl_name` for consistency.